addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/RFXComHandlerFactory.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal;
+
+import java.util.Hashtable;
+import java.util.Set;
+
+import org.openhab.binding.rfxcom.RFXComBindingConstants;
+import org.openhab.binding.rfxcom.discovery.RFXComDeviceDiscoveryService;
+import org.openhab.binding.rfxcom.handler.RFXComBridgeHandler;
+import org.openhab.binding.rfxcom.handler.RFXComHandler;
+import org.osgi.framework.ServiceRegistration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.eclipse.smarthome.config.discovery.DiscoveryService;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+
+import com.google.common.collect.Sets;
+
+/**
+ * The {@link RFXComHandlerFactory} is responsible for creating things and thing 
+ * handlers.
+ * 
+ * @author Pauli Anttila - Initial contribution
+ */
+public class RFXComHandlerFactory extends BaseThingHandlerFactory {
+    
+	@SuppressWarnings("unused")
+	private Logger logger = LoggerFactory.getLogger(RFXComHandlerFactory.class);
+	private ServiceRegistration<?> discoveryServiceReg;
```

Comment:
You need a Set here as you need to keep track of a discovery service for each rfxcomBridgeHandler